libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/DialogManager.cs

```diff
@@ -84,6 +86,7 @@ public Dialog RootDialog
                     this.rootDialogId = value.Id;
                     this.Dialogs.TelemetryClient = value.TelemetryClient;
                     this.Dialogs.Add(value);
+                    this.RegisterDefaultLanguagePolicy(value);
```

Comment:
> this.RegisterDefaultLanguagePolicy(value); [](start = 19, length = 43)

We can't do this here, dialogManager is about to go into Dialogs assembly, doesn't have access to the policy.

Reply:
yeah, make sense, i do it here because i need to make sure the policy i share is come from root dialog, how i can do that outside?